site/content/blog/2020-05-24-zap-2-9-0-highlights/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "ZAP 2.9 Highlights"
+description: "Highlights from ZAP release 2.9.0"
+images:
+- https://www.zaproxy.org/blog/2020-05-24-zap-2-9-0-highlights/images/zapbot.png
```

Comment:
It'd be nice to use something different here. I don't have any ideas though. Suggestions please?

Reply:
@thc202 suggested that we might have an image of zapbot with a package.

Reply:
I searched various images types in the zaproxy-website repo and don't see one with a package. @psiinon do you have other zapbot images somewhere?